mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -136,3 +138,5 @@ CLASS net/minecraft/class_3222 net/minecraft/server/network/ServerPlayerEntity
 		ARG 1 world
 		ARG 2 centerPos
 	METHOD method_30631 getSpawnAngle ()F
+	METHOD method_7346 closeHandledScreen ()V
+		COMMENT Closes the current handled screen and sends a screen closing packet to the client.
```

Comment:
Note that this is inherited from `PlayerEntity`, but I added some javadoc specific to `ServerPlayerEntity`.